examples/sn-blog-gatsby/gatsby-config.js

```diff
@@ -1,3 +1,8 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fetch = require('node-fetch')
+const { codeLogin } = require('@sensenet/authentication-oidc-react')
+const { configuration, repositoryUrl } = require('./configuration')
+
 module.exports = {
   siteMetadata: {
     title: 'sensenet blog',
```

Comment:
siteUrl (not shown in the changes) is not a valid url
author url (not show in the changes) should be https